src/temp/ImagesSlider/ImagesSlider.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { ReactElement, useEffect } from 'react';
+import Stories from 'react-insta-stories';
+import getImageUrl from '../../utils/getImageURL';
+import styles from './Slider.module.scss';
+import { SliderImage } from '../../features/projects/components/PlantLocation/ImageSlider';
+
+interface Props {
+  images: SliderImage[];
+  height: number | string;
+  imageSize: string;
+  type: string;
+}
+
+const ImagesSlider = ({ images, height, imageSize, type }: Props) => {
+  const [slider, setSlider] = React.useState<ReactElement>();
+  const projectImages: { content: () => ReactElement }[] = [];
+
+  useEffect(() => {
+    images.forEach((sliderImage) => {
+      if (sliderImage.image) {
+        const imageURL = getImageUrl(type, imageSize, sliderImage.image);
+
+        projectImages.push({
+          content: () => (
+            <div
+              className={styles.sliderContent}
+              style={
+                type === 'coordinate'
+                  ? { background: `url(${imageURL})` }
+                  : {
+                      background: `linear-gradient(180deg, rgba(0, 0, 0, 0.00) 0%, rgba(0, 0, 0, 0.49) 100%),url(${imageURL})`,
+                    }
+              }
+            >
+              <p className={styles.sliderContentText}>
+                {sliderImage.description}
+              </p>
```

Comment:
Generating JSX elements dynamically within the useEffect hook could lead to performance issues, especially if there are many images. Consider optimizing or lazy-loading images if necessary.